svthalia/concrexit
# Fix translations for the payment model

Closes #1018


### Summary
Fix translations for the payment model

### How to test
Steps to test the changes you made:
1. Go to a payment in the admin
2. Check the names of the fields
